### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+A datamover for CSI snapshots
+</div>
+
+Design Proposal (in-progress): https://github.com/openshift/oadp-operator/pull/597/files
+
+###Basic CSI snapshot workflow steps are as follows:
+- Install OADP Operator, have a stateful application running and create a backup, please
+follow the steps specified [here](https://github.com/openshift/oadp-operator/blob/master/docs/examples/csi_example.md)
+- [Install](https://volsync.readthedocs.io/en/stable/installation/index.html) VolSync controller
+- We will be using VolSync's restic option, hence configure a [restic secret](https://volsync.readthedocs.io/en/stable/usage/restic/index.html#id2).
```
Comment (savitharaghunathan) [inline README.md:12]: is it worth mentioning we are expecting the secret gets created with the name - `restic-secret` ?

**Review by savitharaghunathan — COMMENTED** [review]
Just one nit, otherwise lgtm. Thanks @shubham-pampattiwar :)

File: README.md (hunk 1 above)
Comment (shubham-pampattiwar) [reply README.md:12]: `restic-config` you mean ?

Comment (savitharaghunathan) [reply README.md:12]: No, our code expects the secret name as `restic-secret`. Thats why I thought its worth mentioning that.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+The restic secret should be of the name `restic-config`.
```
Comment (savitharaghunathan) [inline README.md:13]: change `restic-config` -> `restic-secret`

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

**Pull request merged**